sale_financial_risk/models/sale.py

```diff
@@ -37,6 +37,13 @@ def action_confirm(self):
                 }).action_show()
         return super(SaleOrder, self).action_confirm()
 
+    @api.model
+    def _get_risk_states(self):
+        risk_states = ['sale']
+        if self.env['ir.config_parameter'].sudo().get_param('sale.auto_done_setting'):
```

Comment:
System parameters usually use the module name as prefix, so I propose instead: `sale_financial_risk.include_done_orders`. Anyway, not sure if this should be a configuration per company instead of at system level.

Reply:
I use an existing parameter https://github.com/odoo/odoo/blob/12.0/addons/sale/models/sale.py#L746 i can make an independent configuration, but i  only see the necessity of add the done sale orders to the risk calculation if auto done is set.

Reply:
OK, then I let original author to decide

Reply:
Since we modify this we can use a new parameter to give more flexibility to the flows.

Reply:
I don't even see the point of this much detail. It should just calculate all sale orders in sale or done state regardless of setting. Whether auto done is enabled or not a user can still lock a sale order without having invoiced it. The 2 states are essentially the same for risk calculation purposes.

Reply:
@gdgellatly this is comented here:
https://github.com/OCA/credit-control/issues/74